static/app/components/performance/searchBar.spec.tsx

```diff
@@ -99,4 +99,47 @@ describe('SearchBar', () => {
     expect(screen.getByText(textWithMarkupMatcher('clients.call'))).toBeInTheDocument();
     expect(screen.getByText(textWithMarkupMatcher('clients.fetch'))).toBeInTheDocument();
   });
+
+  it('Responds to keyboard navigation', async () => {
+    const onSearch = jest.fn();
+    eventsMock = MockApiClient.addMockResponse({
```

Comment:
are you not asserting this only gets called X times, or are you doing just pure UX test? I know it's not part of the UX contract but it's still part of our technical concerns since it's a contract between the FE/BE to not call the api millions of times, I usually still then prefer to include api call counts and shape especially when using things like debounce.

Reply:
The test above this one deals with sending networks requests, so it asserts on the count; this one is just about keyboard navigation, so it doesn't bother